### DISCUSSION TITLE
VLE for Water-CO2 mixtures

### DISCUSSION THREAD

**Discussion by fabianoscheufele** [CoolProp/CoolProp#2300] [Q&A]
Hello, 

I am new to CoolProp, I would like to know if it is possible to calculate properties for a binary mixture of Water and CO2 vapor-liquid equilibria.

For instance, 
**Known**: P and T conditions as well as global molar fraction
**Unknown**: Liquid density, Gas density, composition of CO2 and water in liquid and gas phases (xCO2, xH2O, yCO2, yH2O).

Probably a Flash Algorithm, right?

*The conditions may be in the supercritical condition range for CO2.

If anybody can suggest where I can find information about this subject and how to call the algorithm (in Excel or Python), I really appreciate it!

Thanks!!!

**Comment by ibell** [comment]
Yes and no.  The most accurate model is there,  but this is a complex
system because it has VLE and LLE. The algorithms in CoolProp are not
robust enough to handle all of that complexity.

**Reply by fabianoscheufele** [reply]
Thanks @ibell!
